app/views/lectures/show.html.erb

```diff
@@ -12,6 +12,9 @@
     <a class="nav-link" id="feedback-tab" data-toggle="tab" href="#feedback" role="tab" aria-controls="feedback" aria-selected="false">Feedback</a>
   </li>
   <% if !@current_user.is_student %>
+    <li class="nav-item">
+      <a class="nav-link" id="enrollmentKey-tab" data-toggle="tab" href="#enrollmentKey" role="tab" aria-controls="enrollmentKey" aria-selected="false">Enrollment Key</a>
+    </li>
```

Comment:
Do we really want to have the tab even if we don't have a key for the lecture?

Reply:
keys are currently required

Reply:
Ah, they shouldn't be according to #164 . Sorry, I assumed it was in this story, too, though it isn't.